tests/subworkflows/nf-core/fasta_binning_concoct/main.nf

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FASTA_BINNING_CONCOCT } from '../../../../subworkflows/nf-core/fasta_binning_concoct/main.nf'
+
+workflow test_fasta_binning_concoct {
+
+    ch_fasta = Channel.of(
+            [[ id: 'test' ], file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)],
+            [[ id: 'test2'], file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)]
+        ).dump(tag: "fasta")
+    ch_bam   = Channel.of(
+            [[ id: 'test' ], file(params.test_data['sarscov2']['illumina']['test_single_end_sorted_bam'], checkIfExists: true), file(params.test_data['sarscov2']['illumina']['test_single_end_sorted_bam_bai'], checkIfExists: true)],
+            [[ id: 'test2'], file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true), file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true)]
+        ).dump(tag: "bam")
```

Comment:
I know this can be used, but for consistency with the rest of the repo would it not be better to declare the inputs separately? I guess the reason we adopted the syntax of declaring them before the call to the module is that it is more evident for newbies, though I don't think this is a guideline so up to you 😄

Reply:
I'm not sure I follow, can you re-phrase? (sorry...)

Reply:
Got slack confirmation this was a misunderstanding!

Reply:
Yes sorry